### DISCUSSION TITLE
Other developer platform

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#9] [Q&A] [answered]
Is there any other developer platform where I can find those sources?

**Comment by Stagyrite** [comment, marked as answer]
Yes, it's on [Bitbucket](https://bitbucket.org/Stagyrite/perl-1.0/src/master/). This process of importing Perl 1 even has its history.
Perl 1 was pushed to Bitbucket a long time ago. The author of the push made that particular version better in terms of compiling errors. Then, the repository was removed from Bitbucket, and [people](https://github.com/AnaTofuZ/Perl-1.0/network/members) pushed it to GitHub. That [GitHub](https://github.com/AnaTofuZ/Perl-1.0) repository was left almost intact and [forked](https://github.com/kironia/Perl-1.0) this year. The forked repositories were fixed to compile. Finally, the valid code was imported back to [Bitbucket](https://bitbucket.org/Stagyrite/perl-1.0/src/master/).
